controls/SV-245540.rb

```diff
@@ -35,19 +32,24 @@
   tag cci: ['CCI-001233']
   tag nist: ['SI-2 (2)']
 
-  if virtualization.system.eql?('docker')
+  only_if('Control not applicable within a container', impact: 0.0) do
+    !virtualization.system.eql?('docker')
+  end
+
+  if input('skip_endpoint_security_tool')
     impact 0.0
-    describe "Control not applicable within a container" do
-      skip "Control not applicable within a container"
+    describe 'Implementing the Endpoint Security for Linux Threat Prevention tool is not applicable by agreement with  the approval authority of the organization.' do
+      skip 'Implementing the Endpoint Security for Linux Threat Prevention tool is not applicable by agreement with  the approval authority of the organization.'
     end
   else
-    describe package('mcafeetp') do
+    linux_threat_prevention_package = input('linux_threat_prevention_package')
```

Comment:
I would guess we really should add an input here to 'check_via_service' and 'check_via_process' since if they are using another tool it may use a service vs a process approach.

Reply:
This input should be renamed to process since that is what we are checking